daemon/server/local_scoot_test.go

```diff
@@ -0,0 +1,65 @@
+package server
+
+import (
+	"github.com/scootdev/scoot/runner"
+	"github.com/scootdev/scoot/runner/execer"
+	"github.com/scootdev/scoot/runner/execer/os"
+	"github.com/scootdev/scoot/runner/runners"
+	"github.com/scootdev/scoot/snapshot"
+	"github.com/scootdev/scoot/snapshot/snapshots"
+
+	"strings"
+	"testing"
+)
+
+func TestNoRunners(t *testing.T) {
+
+	var exec execer.Execer = os.NewExecer()
+	var noopCheckouter snapshot.Checkouter = snapshots.MakeNoOpCheckouter()
```

Comment:
We've been preferring shorter names in general since that's golang convention (although I'm not a huge fan of single letter vars). Also ':=' should suffice here.